depot-tools-git/PKGBUILD

```
@@ -71,5 +59,7 @@ package() {
 	mv "$pkgdir/opt/depot_tools/gclient_completion.sh" "$pkgdir/usr/share/bash-completion/completions/gclient"
 	mv "$pkgdir/opt/depot_tools/git_cl_completion.sh" "$pkgdir/usr/share/bash-completion/completions/git-cl"
 
-	# TODO: Install zsh-goodies/?
+	# Install zsh completions
+	install -d "$pkgdir/usr/share/zsh/plugins/"
+	mv "$pkgdir/opt/depot_tools/zsh-goodies" "$pkgdir/usr/share/zsh/plugins/"
```

Why install this as a plugin and make the user deal with sourcing it rather than just installing it as a completion and moving on?

```suggestion
	install -Dm0644 -t "$pkgdir/usr/share/zsh/site-functions/" "$pkgdir/opt/depot_tools/zsh-goodies/_gclient"
```

---

I wasn't aware that was a thing. I was basing off of were zsh-autosuggestions would install.

---

for 
```bash
install -d "$pkgdir/usr/share/bash-completion/completions/"
	mv "$pkgdir/opt/depot_tools/gclient_completion.sh" "$pkgdir/usr/share/bash-completion/completions/gclient"
	mv "$pkgdir/opt/depot_tools/git_cl_completion.sh" "$pkgdir/usr/share/bash-completion/completions/git-cl"
```
what if we change it too something like this?
```
	install -Dm0644 -t "$pkgdir/opt/depot_tools/gclient_completion.sh" "$pkgdir/usr/share/bash-completion/completions/gclient"
	install -Dm0644 -t "$pkgdir/opt/depot_tools/git_cl_completion.sh" "$pkgdir/usr/share/bash-completion/completions/git-cl" 
```
and doing the same for the others.

---

Yes please, but that code is broken, you need to loose the `-t` in that usage.

---

Ah, ok. doesn't seem to work for bash completions only tho, unfortunately.
I was hopping it work for the other files too, but doesn't seems to be the case.

---

The `install` program handles files (one by one with the option to rename as in these bash ones or in bulk to a target directory with `-t`) but not directories without enumerating all the contents; `cp` or `mv` are needed for directories, although it is best to avoid that when possible. Sometimes it is unavoidable.